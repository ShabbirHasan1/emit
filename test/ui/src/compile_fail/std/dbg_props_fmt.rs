fn main() {
    emit::dbg!(#[emit::fmt("#?")] a: 42);
}
