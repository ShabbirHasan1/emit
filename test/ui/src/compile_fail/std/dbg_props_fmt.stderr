error: `fmt` isn't valid here; it can only be applied to values in templates or event macros
 --> src/compile_fail/std/dbg_props_fmt.rs:2:16
  |
2 |     emit::dbg!(#[emit::fmt("#?")] a: 42);
  |                ^^^^^^^^^^^^^^^^^^^^^^^^
